vulnerability_report.py

```
@@ -0,0 +1,432 @@
+    ax.set_ylim(top=100)
+    ax.set_xlabel("Days")
+    ax.set_title("Fixed in day(s)")
+    save_graph(fig, prefix, "days", svg_output, png_output)
+
+
+def generate_vulnerable_packages_graph(
+    stat: dict[str, Any], prefix: str, svg_output: bool, png_output: bool
+) -> None:
+    fig, ax = plt.subplots()
+    D = stat["packages"]
+    names, counts = zip(*D.most_common(10))
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)